### DISCUSSION TITLE
Managing Access tokens while login

### DISCUSSION THREAD

**Discussion by ghbishal** [obytes/react-native-template-obytes#256] [Q&A] [answered]
I was trying to handle access tokens that i get during the login. 

Currently inside `core/auth/utils` you have `const TOKEN = 'token' ` also you have made TokenType which consists of access and refresh. how can i handle that part ?

I want to automatically add the all tokens directly form the api call most important thing. if the user is not using app for like 24 days token should automatically be removed and user should again login.

also in the LoginForm onSubmit

there is `access-token` , `refresh` how do i handle that? could you please help me out?

**Comment by yjose** [comment, marked as answer]
You can read [Authentication docs](https://starter.obytes.com/guides/authentication/) for more details. but overall `TOKEN` is mainly a name to save tokens in the mmkv storage
